admin/config.js

```diff
@@ -18,6 +18,9 @@ module.exports = {
   TSO_AUTH_PUBLIC_KEY: process.env.TSO_AUTH_PUBLIC_KEY,
   TIME_BETWEEN_QUESTIONS: 2,
   LINES_PER_CHECK_FORM: 25,
+  Data: {
+    allowedWords: process.env.ALLOWED_WORDS || 'ant,bag,ben,bid,bin,box'
```

Comment:
@jon-shipley I wanted to provide some default values in case the env is not definedas well but I was under the impression we wanted to find a solution that includes travis.
As far as I tried there was no way for travis to expose node env variables.